app/javascript/react/components/ReviewTile.test.js

```diff
@@ -0,0 +1,30 @@
+import React from "react"
+import Enzyme, { mount } from "enzyme"
+import Adapter from "enzyme-adapter-react-16"
+import { BrowserRouter } from 'react-router-dom'
+Enzyme.configure({ adapter: new Adapter() })
+
+import ReviewTile from "./animals/ReviewTile"
+
+describe("ReviewTile", () => {
+  let wrapper
+
+  beforeEach(() => {
+    wrapper = mount(
+      <BrowserRouter>
+        <ReviewTile
+          key="1"
+          id="1"
+          title="cute elephant"
+          rating="5"
+          review_body="Cute elephant"
+        />
+      </BrowserRouter>
+    )
+  })
+
+  it("renders an p tag with the reviews", () => {
+    expect(wrapper.find("h4").text()).toBe("Title: cute elephant")
```

Comment:
your `it` block says "renders a p tag" but your test is looking for an `h4` tag. While the `it` is strictly informational, it is important that they match so that when looking at the test later you don't have to wonder which one it is supposed to be doing.